setup.sh

```diff
@@ -32,18 +58,36 @@ SHARE="/scratch/users/vsochat/share"
 echo "A containershare (https://vsoch.github.io/containershare is a library of
 containers that are prebuilt for you, and provided on your cluster resource. if you
 are at Stanford, leave this to be the default. If not, ask your HPC administrator
-about setting one up, and direct them to https://www.github.com/vsoch/containershare."
+about setting one up, and direct them to https://www.github.com/vsoch/containershare.
+For farmshare, leave blank to use default singularity maintained by Paul Nuyujukian (/farmshare/home/classes/bioe/301p/ce/ces)"
 echo
 read -p "container shared folder (default for Stanford: ${SHARE}) > " CONTAINERSHARE
-CONTAINERSHARE=${CONTAINERSHARE:-${SHARE}}
-
+if $SHERLOCK
+then
+   CONTAINERSHARE=${CONTAINERSHARE:-${SHARE}}
+else
+   CONTAINERSHARE=${CONTAINERSHARE:-/farmshare/home/classes/bioe/301p/ce/ces}
+fi
 echo
 
 MEM=20G
 
 TIME=8:00:00
 
-for var in FORWARD_USERNAME PORT PARTITION RESOURCE MEM TIME CONTAINERSHARE
+echo 
+echo "Finally, please supply the directory at which the lsof command is located. This is used to kill ports that the we are forwarding when
```

Comment:
Sorry this isn't what I had in mind either. The directory of lsof is very unlikely to change. We can also just create logic that automatically sets USE_LSOF to true given sherlock and we don't have to ask anything. This should just be a boolean value that is automatically set.

Reply:
Fixed and better setup.sh code is where USE_LSOF and other variables are better defined in default.